service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -279,29 +282,47 @@ public static JudgeAndLegalAdvisor createJudgeAndLegalAdvisor(final String legal
             .build();
     }
 
-    public static List<Element<Direction>> createDirection(DirectionAssignee assignee) {
+    public static List<Element<Direction>> createElementCollection(Direction direction) {
         return ImmutableList.of(
             Element.<Direction>builder()
-                .value(Direction.builder()
-                    .directionText("Mock direction text")
-                    .assignee(assignee)
-                    .build())
-                .build());
+                .value(direction)
+                .build()
+        );
     }
 
-    public static List<Element<Direction>> createCustomDirection(DirectionAssignee assignee) {
-        return ImmutableList.of(
-            Element.<Direction>builder()
-                .value(Direction.builder()
-                    .directionText("Mock direction text")
-                    .assignee(assignee)
-                    .readOnly("No")
-                    .custom("Yes")
-                    .build())
-                .build());
+    public static Direction createDirection(DirectionAssignee assignee) {
+        return Direction.builder()
+            .directionText("Mock direction text")
```

Comment:
This is class in `src/main` so using `Mock direction text` does not look right. Is that test helper? Shall it be moved to `src/test`?

Reply:
Good point, yeah this is largely been used for generating data to assist with tests. For now can I rename the direction text to not imply that it's to be used solely for mocking and we will move in another PR?

Reply:
I am OK with that as long as you clean it up immediately after this PR is merged to master.